src/util/Util.js

```diff
@@ -267,6 +267,22 @@ class Util {
     return { animated: Boolean(m[1]), name: m[2], id: m[3] || null };
   }
 
+  /**
+   * Resolves a partial emoji object from an {@link EmojiIdentifierResolvable}, without checking a Client.
+   * @param {EmojiIdentifierResolvable} emoji Emoji identifier to resolve
+   * @returns {Object} Object with possible `animated`, `name`, and `id` properties
```

Comment:
Since it's returning `null` when resolving fails.
```suggestion
   * @returns {?RawEmoji} Object with possible `animated`, `name`, and `id` properties
```
Shouldn't this also use the new typdef? (The description is redundant then too)